Formula/luaradio.rb

```diff
@@ -27,7 +28,21 @@ def install
         s.gsub! "$(DESTDIR)$(INSTALL_CMOD)/radio.so",
                 "$(PREFIX)/lib/lua/5.1/radio.so"
       end
-      system "make", "install", "PREFIX=#{prefix}"
+      system "make", "install", "PREFIX=#{libexec}"
+
+      env = {
+        PATH:      "#{Formula["luajit-openresty"].opt_bin}:$PATH",
+        LUA_PATH:  "#{libexec}/lib/lua/5.1/?.lua${LUA_PATH:+:$LUA_PATH}",
+        LUA_CPATH: "#{libexec}/lib/lua/5.1/?.so${LUA_CPATH:+:$LUA_CPATH}",
+      }
+
+      Dir["#{libexec}/bin/*"].each do |path|
```

Comment:
Isn't this the same as `env_script_all_files` or whatever that one is called?

Reply:
Looks like it; I just based this off another formula. Let me fix that.

Reply:
Actually, it's not quite the same. I'll need to change other parts of the install block to use this. Might break the binaries if relative paths are baked in. Still, worth a try.